src/services/auth.js

```diff
@@ -0,0 +1,21 @@
+// TODO: Use typedi or similar
+const Account = require("../models/Account");
+
+class AuthService {
+    constructor() {
+
+    }
+
+    async register(username, password) {
+        const account = await Account.create({
+            username: username,
+            password: password,
+        });
+
+        return {
+            username: account.username,
```

Comment:
We can use `{ username }` directly, since it will be the same as `account.username`

Reply:
Ideally so, but some hooks in the Model might change it, due to trimming or something of the sort. I see no problem with leaving this as is.